sklearn/utils/estimator_checks.py

```diff
@@ -67,7 +67,17 @@
                 'RandomForestRegressor', 'Ridge', 'RidgeCV']
 
 
-def _yield_non_meta_checks(name, Estimator):
+def assert_almost_equal_dense_sparse(x, y, decimal=6, err_msg=''):
+    if sparse.issparse(x):
+        assert_array_almost_equal(x.data, y.data,
```

Comment:
TBH, I've seen it said somewhere. And I can't recall the arguments put forth, but I like the brief name, and I like that we're able to use `rtol` when appropriate, and it's good practice to be thinking about whether absolute or relative tolerance is the issue for a particular case.